### PR TITLE
:white_check_mark: update e2e tests (new explore)

### LINKED ISSUE #4692: Update e2e test for new explore
Screenshot 2023-01-13 at 23 14 33

### PR DESCRIPTION
**Pull request by petersopko**
## PR Type

- [ ] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #4692 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by petersopko** [comment]
What was done here:

- moving away from having one large E2E file, expand parallelization by 1 (self-explanatory)
- moving away from using commands as function store; this was a mistake on my part in the past
- update explorer tests to reflect the newest changes

What can be done to improve (I will continue on this in the new PR as I'm planning to revisit tests in general): 

- there are at least two ugly dropdown selectors; I couldn't figure it out in a neat way
- finish moving away from commands as function store, split the basicE2E test even further

File: tests/cypress/e2e/explore.cy.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
+describe('exploreRmrkCollections', () => {
```
[the remark this answers is not in the document]
Comment (petersopko) [reply tests/cypress/e2e/explore.cy.ts]: done

**Review by roiLeo — APPROVED** [review]
✅ 🧪

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 31.18 USD/KSM ~ 1.604 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0xc8b7f783321790435a12b831eb7c2a7d606f4e7381704361bfae9985ba146c44](https://kusama.subscan.io/extrinsic/0xc8b7f783321790435a12b831eb7c2a7d606f4e7381704361bfae9985ba146c44)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**